package/src/components/MessageInput/FileUploadPreview.tsx

```diff
@@ -162,7 +182,6 @@ const FileUploadPreviewWithContext = <
             removeFile(item.id);
           }}
           style={[styles.dismiss, { backgroundColor: overlay }, dismiss]}
-          testID='remove-file-upload-preview'
```

Comment:
Let's double check that this doesn't break some test

Reply:
✅